src/renderer/src/applets/applet-host.ts

```diff
@@ -355,6 +355,9 @@ export async function handleAppletIframeMessage(
           (profile) => !!profile,
         ) as GroupProfile[];
 
+        // Extract the group DNA hash (there's always exactly one for applet-view)
```

Comment:
I don`t believe this is true, since the IframeConfig is also storing multiple groupProfiles.

`groupStore` is the first one because its just for retrieving the authentificationToken and its possibility the same for every group.